src/Datadog.Trace/Tracer.cs

```diff
@@ -353,6 +355,18 @@ void IDatadogTracer.Write(Span[] trace)
             }
         }
 
+        /// <summary>
+        /// Reports the detected version of the agent
+        /// </summary>
+        /// <param name="version">Version of the agent</param>
+        void IDatadogTracer.ReportAgentVersion(string version)
+        {
+            if (ShouldLogPartialFlushWarning(version))
+            {
+                Log.Warning("DATADOG TRACER DIAGNOSTICS - Partial flush should only be enabled with agent 7.26.0+ (detected version: {version})", version);
```

Comment:
Isn't this going to say `(detected version: )` with older Agent versions?

Reply:
Yeah, but we know what it means. Do you prefer that I put a placeholder value?

Reply:
Should we remove `(detected version: {version})` if it's never going to show a version?
```suggestion
                Log.Warning("DATADOG TRACER DIAGNOSTICS - Partial flush should only be enabled with Agent 7.26.0+", version);
```

Reply:
Fixed in https://github.com/DataDog/dd-trace-dotnet/pull/1347/commits/b43b7e20436d6690d745020454ad8422655ae030